src/components/TextField.spec.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint no-console: 0  */
+import React from 'react';
+import { spy } from 'sinon';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import TextField from './TextField.jsx';
+
+const onNewValueSpy = spy();
+
+const initProps = {
+  title: 'title',
+  tag: 'test-tag',
+  hide: true,
+  error: '',
+  currentValue: 'value',
+  onNewValue: onNewValueSpy,
+};
+
+describe('TextField.jsx', () => {
+  it('should call the handler function that was passed as a prop', () => {
+    const wrapper = shallow(<TextField {...initProps} />);
+    wrapper.find('#test-tag').simulate('change', { target: { value: 'Changed Value' } });
+    expect(onNewValueSpy.calledOnce).to.equal(true);
+  });
+
+  it('should display the FormHelperText component when there is no error', () => {
```

Comment:
This and the following test seem contradictory

Reply:
yeah that was a typo, fixed